arbeitszeit_web/colors.py

```diff
@@ -0,0 +1,26 @@
+from typing import Dict, Protocol
+
+
+class Colors(Protocol):
+    @property
+    def primary(self) -> str:
+        ...
+
+    @property
+    def info(self) -> str:
+        ...
+
+    @property
+    def warning(self) -> str:
+        ...
+
+    @property
+    def danger(self) -> str:
+        ...
+
+    @property
+    def success(self) -> str:
+        ...
+
+    def get_dict(self) -> Dict[str, str]:
```

Comment:
I think that this method deserves a better name. I read `Colors.get_dict` and have no idea what this method is supposed to do.